### DISCUSSION TITLE
HTTP calls to perform action

### DISCUSSION THREAD

**Discussion by sengduong** [babybuddy/babybuddy#582] [Ideas]
Use case: Setup RFID/NFS tags with predefined URLs/actions, which can be used with a smartphone to perform a action. e.g. add entry to BabyBuddy. 

I know this could be achived with home-assistant, but I would prefer a more stand-alone solution to have a more "portable" solution which can be handed over to family/friends taking care of the baby.

**Comment by cdubz** [comment]
I think Baby Buddy's existing API should support this -- check the the [API documentation](https://docs.baby-buddy.net/api/). Any functionality missing for this use case?
